CONTRIBUTING.md

```diff
@@ -52,13 +53,28 @@ Any comment without `[noblock]` included in it will block AutoMerge from automat
     * Sometimes package names are possibly confusing or don't conform to our [naming guidelines](naming-guidelines), but AutoMerge does not detect this.
     Feel free to comment, describing what you think is confusing or non-compliant about the current name, and any suggestions you have for a more clear name.
 
+### Other PRs to General
+
+Sometimes, the registry needs to be updated in other ways that involve manual pull requests (PRs) rather than auto-generated ones.
+The most common reason is to update the URL for a repository.
+
+#### Updating the URL for a repository
+
+If someone transfers a GitHub repository, [we ask](https://github.com/JuliaRegistries/General#how-do-i-transfer-a-package-to-an-organization-or-another-user) that they update the URL stored in General.
+This is done by manually making a PR to General to update the URL.
+You can review such a PR by checking that the old URL redirects to the new one.
+* If it does, that's a clear sign that the change is legitimate and the new URL is correct.
+  If you have write permissions to General, you can merge the PR; otherwise you can approve it or comment.
+* If it does not, you can ask the author why. This should be handled on a case-by-case basis.
```

Comment:
Explain that this requires to make sure all registered revisions (that is, their git tree sha1s) are accessible?

Reply:
We could probably automate that right? Could run a workflow on every non-automated PR, check if a url changed, and if so check if all the versions exist.

Reply:
But yea for this PR I should at least mention it

Reply:
If that's doable, that'd be helpful!

Reply:
ok, I mentioned this + checking that the change is legit